x/bank/internal/keeper/keeper.go

```diff
@@ -321,6 +327,12 @@ func (keeper BaseSendKeeper) SetSendEnabled(ctx sdk.Context, enabled bool) {
 	keeper.paramSpace.Set(ctx, types.ParamStoreKeySendEnabled, &enabled)
 }
 
+// BlacklistedAddrs returns the list of the blacklisted accounts addresses that
+// are not allowed to receive funds
+func (keeper BaseSendKeeper) BlacklistedAddrs() map[string]bool {
```

Comment:
An upstream caller will actually be able to mutate this map because maps are passed by reference! Instead of `BlacklistedAddrs`, let's implement a method `BlacklistedAddr(addr sdk.AccAddress) bool`. If `true`, return error on sends.